### PR TITLE
270-edit-phone-number-from-profile

### PR DESCRIPTION
**Pull request by serhiibets**

### CODE REVIEW

File: wayat/frontend/ios/firebase_app_id_file.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (capf4bbo) [inline wayat/frontend/ios/firebase_app_id_file.json:1]: This file has critical data, please delete it and add it to the frontend gitignore

File: wayat/frontend/ios/Runner/AppDelegate.swift [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+import GoogleMaps
 
 @UIApplicationMain
 @objc class AppDelegate: FlutterAppDelegate {
   override func application(
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
+    
   ) -> Bool {
+    GMSServices.provideAPIKey("API-KEY")
```
Comment (capf4bbo) [inline wayat/frontend/ios/Runner/AppDelegate.swift:12]: Here you need to pass the API-KEY from the environment variables, it should be called GOOGLE_MAPS_KEY

**Review by capf4bbo — CHANGES_REQUESTED** [review]

File: wayat/frontend/ios/firebase_app_id_file.json (hunk 1 above)
Comment (serhiibets) [reply wayat/frontend/ios/firebase_app_id_file.json]: Fix

File: wayat/frontend/ios/Runner/AppDelegate.swift (hunk 2 above)
Comment (serhiibets) [reply wayat/frontend/ios/Runner/AppDelegate.swift:12]: fix

**Pull request closed without merging**